src/api/attribute/service.ts

```diff
@@ -118,7 +118,6 @@ async function list (attributesParam: AttributeListParam, config, indexName: str
   try {
     const query = adjustQuery({
       index: indexName,
-      type: 'attribute',
```

Comment:
i found this exact problem aswell yesterday! great work!

Reply:
there are also other places where adjustQuery gets called with a type... for example:

src\api\extensions\elastic-stock\index.js in line 16...

idea... how about we change adjustQuery to something like this:

```
function adjustQuery (esQuery, entityType, config) {
  if (parseInt(config.elasticsearch.apiVersion) < 6) {
    esQuery.type = entityType
  } else {
    delete esQuery.type
  }
  esQuery.index = adjustIndexName(esQuery.index, entityType, config)
  return esQuery
}
```

@cewald @gibkigonzo

Reply:
Good idea, I've added it.